lib/Classifiers/Classifier.php

```diff
@@ -1,107 +1,91 @@
 <?php
-/*
- * Copyright (c) 2021. The Nextcloud Recognize contributors.
- *
- * This file is licensed under the Affero General Public License version 3 or later. See the COPYING file.
- */
 
```

Comment:
```suggestion
// SPDX-FileCopyrightText: 2021 The Nextcloud Recognize contributors
// SPDX-License-Identifier: AGPL-3.0-or-later
```

instead of deleting the copyright, can be replaced by a concise SPDX/reuse statement

Reply:
Oh, I didn't actually intend to remove the copyright. Added it back in :)